qiime2/plugin/plugin.py

```diff
@@ -57,9 +58,11 @@ def __init__(self, name, version, website, package, citation_text=None,
             self.description = description
 
         if citations is None:
-            self.citations = ()
-        else:
-            self.citations = tuple(citations)
+            citations = ()
+
+        self.citations = Citations([
+            ('plugin|%s:%s|%d' % (self.name, self.version, idx), entry)
```

Comment:
Not in love with this, but I didn't have a "great" option either, so this seems like the least damage.